src/applications/accreditation/21a/components/HasAConviction.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const HasAConviction = () => (
+  <>
+    This includes felonies, firearms, or explosive violations, misdemeanors, and
+    all other offenses.
+    <br />
+    <br />
+    This does not include:
```

Comment:
There's duplication here and in `IsUnderCharges`. Is a separate component needed to work with the forms library?